src/config/ServerlessLayersConfig.js

```diff
@@ -1,6 +1,21 @@
 class ServerlessLayersConfig {
-  constructor(options) {
+  constructor(options, slsVersion) {
+    console.log(`[ LayersPlugin ]: - options are: ${JSON.stringify(options)}, sls version is ${slsVersion}`);
+
+    if (slsVersion.startsWith('3') && Object.entries(options).length > 0) {
+      const v3Options = {};
+      options.param.forEach((v3Option) => {
+        console.log(`option of v3 is: ${v3Option}`);
+        const v3OptionSplitArr = v3Option.split('=');
+        // @ts-ignore
+        // eslint-disable-next-line prefer-destructuring
+        v3Options[v3OptionSplitArr[0]] = v3OptionSplitArr[1];
+      });
+      options = v3Options;
```

Comment:
```suggestion
      options = Object.entries(options).reduce((res, curr) => {
           const [key, value] = curr;
           res[key] = value;
           return res;
      }, {})
```

suggestion for cleaner code.. double check me it works...

Reply:
Used the reduce, the --param="key=value" still needs to be parsed... And the options.param is an array I'll just reduce it